.gitlab/e2e/e2e.yml

```diff
@@ -101,6 +101,10 @@ k8s-e2e-otlp-main:
     E2E_OUTPUT_DIR: $CI_PROJECT_DIR/e2e-output
   script:
     - inv -e new-e2e-tests.run --targets $TARGETS -c ddagent:imagePullRegistry=669783387624.dkr.ecr.us-east-1.amazonaws.com -c ddagent:imagePullUsername=AWS -c ddagent:imagePullPassword=$(aws ecr get-login-password) --junit-tar "junit-${CI_JOB_NAME}.tgz" ${EXTRA_PARAMS}
+  after_script:
+    # Upload generated junit files
+    - export DATADOG_API_KEY=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh ci.datadog-agent.datadog_api_key_org2)
+    - for f in junit-new-e2e-*.tgz; do inv -e junit-upload --tgz-path "$f"; done
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is this limited to the current job's artifacts?

Reply:
as it's part of the template, it will be executed in all `new-e2e-*` job, so on current job's artifact yes. I'm not sure if I answer correctly your question

Reply:
Can we have more than one `tgz` per job? Looks like we have one, what about calling it `junit.tgz` both here and at line `121`?